tests/test_summary_eval_supabase.py

```diff
@@ -0,0 +1,11 @@
+def test_summary_eval_supabase(client):
+    response = client.post(
+        "/score/summary",
+        json={
+            "textbook_name": "macroeconomics-2e",
+            "chapter_index": "1",
+            "section_index": "1",
+            "summary": """Think about it this way: In 2015 the labor force in the United States contained over 158 million workers, according to the U.S. Bureau of Labor Statistics. The total land area was 3,794,101 square miles. While these are certainly large numbers, they are not infinite. Because these resources are limited, so are the numbers of goods and services we produce with them. Combine this with the fact that human wants seem to be virtually infinite, and you can see why scarcity is a problem.',""",  # noqa: E501
+        },
+    )
+    assert response.json()["containment"] > 0.9
```

Comment:
Unless we're specifically testing containment?

```suggestion
    assert response.status_code == 200
```

Reply:
This was copied from an old unit test. I agree that it makes more sense to assert the status code here.